test/utils/shippable/shippable.sh

```diff
@@ -17,6 +17,15 @@ for container in $(docker ps --format '{{.Image}} {{.ID}}' | grep -v '^drydock/'
     docker rm -f "${container}" || true  # ignore errors
 done
 
+for i in network windows sanity; do
+    if [[ "${script}" == "${i}" ]]; then
```

Comment:
Rather than a loop this can be done with a regexp comparison. I'm not sure if we need to anchor it or not but here's an anchored example.
```suggestion
    if [[ "${script}" =~ ^(network|windows|sanity)$ ]]; then
```